### ISSUE TITLE
Favicon feature added a content disposition attachment header that's wrong

### ISSUE THREAD

**Issue by Rudd-O** [plone/Products.CMFPlone#3419]
Recently-added feature produces a custom favicon that is simply delightful to manage.

However, two things pop to mind:

First, the icon is served as a downloadable file (it shouldn't -- it's intended to be viewed inline, and the header may cause a highly annoying browser download window to pop up in some browsers):

```
curl -v https://example.com/favicon.ico
...
< content-disposition: attachment; filename*=UTF-8''favicon.ico
```

Second, the icon is served under the cache rule `plone.content.file`, which means the file is expired upon serve, which of course in turn means that pretty much every browser will not cache the favicon.  This may have a downstream impact on the site branding of all Plone users using this feature.

My Plone knowledge does not suffice to fix this issue, so I would encourage someone to help me figure out a fix for this.

**Comment by Rudd-O** [comment]
Ah I think what needs to be done is simply to inherit from `DisplayFile` instead of from `Download` in `Products/CMFPlone/browser/favicon.py:class SiteFavicon`.  That ought to fix the first issue.

**Comment by Rudd-O** [comment]
Will be soon fixed by #3418.
